tests/test-webapps/test-servlet-spec/test-spec-webapp/pom.xml

```diff
@@ -219,11 +219,5 @@
       <artifactId>test-container-initializer</artifactId>
       <version>${project.version}</version>
     </dependency>
```

Comment:
Why has the older version of the util jar been removed?

Reply:
I think this was something @joakime merged into the branch... it is an unrelated change.  I'll remove